marl_eval/json_tools/json_utils.py

```diff
@@ -135,28 +136,42 @@ def pull_neptune_data(
     if not os.path.exists(store_directory):
         os.makedirs(store_directory)
 
+    # Suppress neptune logger
+    neptune_logger = logging.getLogger('neptune')
+    neptune_logger.setLevel(logging.ERROR)
+
+    # Initialise a counter to ensure unique file names
+    counter = 0
+
     # Download and unzip the data
     for run_id in tqdm(run_ids, desc="Downloading Neptune Data"):
```

Comment:
This is just for manually controlling the progress bar so that we can use `enumerate` instead of using a loop with the counter. 
```suggestion

    # Download and unzip the data
    pbar = tqdm(total=len(run_ids), desc="Downloading data from Neptune")

    for i, run_id in enumerate(run_ids):
        pbar.update(1)
```